wp-web-push/wp-web-push-db.php

```diff
@@ -56,6 +56,7 @@ public static function on_activate() {
     // Set default options.
     update_option('webpush_title', 'blog_title');
     update_option('webpush_min_visits', 3);
+    update_option('webpush_triggers', array());
```

Comment:
We *could* default to `'new-post'`, we'll need to pull `$ALLOWED_TRIGGERS` into a file shared across the other plugin PHP files.